docs/source/config.json

```diff
@@ -18,6 +18,7 @@
     "Fetching Data": {
       "Resolvers": "/data/resolvers",
       "Fetching from REST": "/data/fetching-rest",
+      "Fetching Data": "/data/fetching-data",
```

Comment:
I think this is a more generic article that should be listed before the REST counterpart

Reply:
I like that! I'll switch them :)